File: app/src/main/java/com/example/sena/MainActivity.java
package com.example.sena;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.google.android.material.button.MaterialButton;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        MaterialButton btnIniciar = findViewById(R.id.btnIniciar);
        MaterialButton btnRegistrarse = findViewById(R.id.btnRegistrarse);

        btnIniciar.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, InicioSesion.class);
            startActivity(intent);
        });

        btnRegistrarse.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, Registro.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/example/sena/MainActivity.kt
package com.example.sena

import android.content.Intent
import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.button.MaterialButton

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        val btnIniciar = findViewById<MaterialButton>(R.id.btnIniciar)

        btnIniciar.setOnClickListener {
            val intent = Intent(this, InicioSesion::class.java)
            startActivity(intent)
        }

        val btnRegistrarse = findViewById<MaterialButton>(R.id.btnRegistrarse)

        btnRegistrarse.setOnClickListener {
            val intent = Intent(this, Registro::class.java)
            startActivity(intent)
        }

    }
}
